Xamarin.ExposureNotification/ExposureInfo.shared.cs

```diff
@@ -32,16 +33,28 @@ public ExposureInfo(DateTime timestamp, TimeSpan duration, int attenuationValue,
 
 	public class ExposureDetectionSummary
 	{
+		[EditorBrowsable(EditorBrowsableState.Never)]
 		public ExposureDetectionSummary(int daysSinceLastExposure, ulong matchedKeyCount, byte maximumRiskScore)
```

Comment:
We can't obsolete these because if I have a `byte`, then it will always pick this one - and then warn. Unless a byte is bad and we need to force the user to explicitly go `int`. However, this is the least annoying  because if I have a byte, then suddenly I get a warning for a member that I never saw.